src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/integTests/framework/Cluster.java

```diff
@@ -273,6 +273,10 @@ public JsonElement getAllRcaDataOnHost(HostTag hostTag, String rcaName) {
     return tagToHostMapping.get(hostTag).getDataForRca(rcaName);
   }
 
+  public <T> T getAllClassDataOnHost(HostTag hostTag, Class<T> className) throws Exception {
```

Comment:
Maybe we can name it something like `read/constructObjectFromDB` ?

Reply:
Using constructObjectFromDB